.gitmodules

```diff
@@ -10,3 +10,6 @@
 [submodule "extern/memory_backend/analytical"]
 	path = extern/remote_memory_backend/analytical
 	url = git@github.com:astra-sim/astra-memory-analytical.git
+[submodule "extern/network_backend/ns3-interface"]
```

Comment:
Why do we use "ns3-interface" as the name instead of ns3?

Reply:
Good point. i have changed this to just use `ns3`.